addons/l10n_es_edi_tbai/models/account_edi_format.py

```diff
@@ -432,7 +432,8 @@ def _l10n_es_tbai_get_invoice_line_price_total(self, invoice_line):
     def _l10n_es_tbai_get_importe_desglose(self, invoice):
         com_partner = invoice.commercial_partner_id
         sign = -1 if invoice.move_type in ('out_refund', 'in_refund') else 1
-        if com_partner.country_id.code in ('ES', False) and not (com_partner.vat or '').startswith("ESN"):
+        if (com_partner.country_id.code in ('ES', False) and not (com_partner.vat or '').startswith("ESN")) \
+                or invoice.l10n_es_is_simplified:
```

Comment:
@LouisGobert I am wondering why the runbot is not failing (maybe the test is left out still), butthe field l10n_es_is_simplified does not exist in 16 yet if I checked correctly.  You could use the _is_l10n_es_tbai_simplified method.